go.sum

```diff
@@ -86,6 +86,8 @@ github.com/dell/gonvme v1.0.0 h1:gd6afy0Hb/HaszG1d6R/nCSShn0TslNkcu8gAeJPDaM=
 github.com/dell/gonvme v1.0.0/go.mod h1:JXLmgxE/2dsiXqW5E+kH5Zo11lOjCuHyiHu92xqR9O0=
 github.com/dell/gopowerstore v1.7.0 h1:5wkYKlgkR5PH7UZO9wN1nBTKF7vlHHWYqVeJN14MY6w=
 github.com/dell/gopowerstore v1.7.0/go.mod h1:rYXLGFxOc0ZIiZkv4S0D7CKW3kaC09y6LKogsVDANJg=
+github.com/dell/gopowerstore v1.7.1-0.20220401105348-104d2bf041c3 h1:SEEEk+A3U2ASnk1NkABjTvvVsBpxm1d48GolLSGuNEE=
+github.com/dell/gopowerstore v1.7.1-0.20220401105348-104d2bf041c3/go.mod h1:rYXLGFxOc0ZIiZkv4S0D7CKW3kaC09y6LKogsVDANJg=
```

Comment:
go mod tidy needed to remove older entries of gopowerstore?

Reply:
Done